hedera-node/configuration/previewnet/application.properties

```diff
@@ -1 +1,4 @@
-
+bootstrap.genesisPublicKey=c249a323c878f5b5e2daccda6d731e6fdc32f870228d1cd4fae559d947dbc36c
+contracts.chainId=297
+ledger.id=0x02
+contracts.evm.version.dynamic=true
```

Comment:
The properties in this file were copied from `hedera-node/configuration/previewnet/bootstrap.properties`.

File `0.0.121` on previewnet appears to be empty